src/main/archetype/ui.permissions/src/main/content/jcr_root/home/groups/__appsFolderName__/custom-asset-admin/.content.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:rep="internal"
+    jcr:mixinTypes="[rep:AccessControllable]"
+    jcr:primaryType="rep:Group"
+    jcr:uuid="a969cdb8-08f4-3158-abb0-c75e65c29f94"
+    rep:authorizableId="custom-asset-admin"
+    rep:principalName="custom-asset-admin">
```

Comment:
See DG's note about using the app folder name for these IDs. `Custom` doens't provide enough context/isn't descriptive enough.

Reply:
It should also be `XXXX-asset-administrators` to stay consistent with naming.